lighthouse-core/audits/dobetterweb/dom-size.js

```diff
@@ -27,8 +27,8 @@ const UIStrings = {
   failureTitle: 'Avoid an excessive DOM size',
   /** Description of a Lighthouse audit that tells the user *why* they should reduce the size of the web page's DOM. The size of a DOM is characterized by the total number of DOM elements and greatest DOM depth. This is displayed after a user expands the section to see more. No character length limits. 'Learn More' becomes link text to additional documentation. */
   description: 'Browser engineers recommend pages contain fewer than ' +
```

Comment:
```suggestion
  description: 'Browser engineers recommend that pages contain fewer than ' +
```

Reply:
(I'm not sure why the diff is showing currently three spaces and this is deleting one when it's clearly going 2->2, but I guess you can't just accept these changes cause it would need collect-strings anyway :)